website/docs/docs/core/source-install.md

```diff
@@ -12,28 +12,58 @@ Downloading and building dbt Core will enable you to contribute to the project b
 
 ### Installing dbt Core
 
-To install `dbt-core` from the GitHub code source:
+Beginning in v1.8, installing an adapter no longer automatically installs `dbt-core`. This is because adapters and dbt Core versions have been decoupled from each other so we no longer want to overwrite existing dbt-core installations
```

Comment:
```suggestion
Beginning in v1.8, installing an adapter does not automatically install `dbt-core`. This is because adapters and dbt Core versions have been decoupled from each other so we no longer want to overwrite existing dbt-core installations
```